src/modules/didRegistry/didRegistry.service.ts

```diff
@@ -199,6 +226,58 @@ export class DidRegistry {
     }
   }
 
+  /**
+   * @description updates public key of currently signed did document
+   * @returns true if document is updated successfuly
+   */
+  async updateSignedDidPublicKey({
```

Comment:
This could be more general if could add key to document of any controlled identity

Reply:
Great point 👍